unstructured/documents/elements.py

```diff
@@ -490,6 +490,8 @@ def to_dict(self) -> dict:
         out["element_id"] = self.id
         out["type"] = self.category
         out["text"] = self.text
```

Comment:
Can we instead add this to the Element class:

embedding: Optional[List[float]] = None


That way we can also have some typing around it.